### PR TITLE
Fix summary for tuples

### PR DESCRIPTION
**Pull request by briochemc**
Removed `sumary(io: IO, t::tuple)` line as suggested in https://github.com/JuliaLang/julia/issues/36251#issuecomment-643320532.

### CODE REVIEW

**Comment by JeffBezanson** [comment]
Thanks. I think there are some tests of this to fix as well.

**Comment by StefanKarpinski** [comment]
The good news is you don't need to add tests, just fix the existing ones. Might be worth a note in NEWS.

**Review by Keno — CHANGES_REQUESTED** [review]
As per previous comments, tests need to be fixed (changing the review status drops it from my queue until comments are addressed).

**Comment by briochemc** [comment]
>  tests need to be fixed

Yes, I'll try to do that early next week, sorry got very little time right now!

**Comment by briochemc** [comment]
I just looked for `summary` in `test/show.jl` and changed those in the last commit (hopefully correctly)... But is that all that was needed?

**Comment by briochemc** [comment]
I *think* I fixed the tests correctly. Please let me know if I need to do anything else!

**Comment by JeffBezanson** [comment]
Looks good, just needs a rebase now.

**Comment by briochemc** [comment]
Again, I *think* I just did the correct thing by merging master back into the PR branch. Do I need to do anything else?

**Comment by KristofferC** [comment]
Looks like some test need fixing up.

File: test/errorshow.jl [hunk 1]
```diff
@@ -291,7 +291,7 @@ let undefvar
     @test occursin("DomainError with [0.0 -1.0 …", err_str)
 
     err_str = @except_str (1, 2, 3)[4] BoundsError
-    @test err_str == "BoundsError: attempt to access (1, 2, 3) at index [4]"
+    @test err_str == "BoundsError: attempt to access Tuple{$Int,$Int,$Int} at index [4]"
```
Comment (JeffBezanson) [inline test/errorshow.jl:294]: Please rebase this (which will hopefully make the test failure go away), and add spaces after these commas (and in the other file as well). Hopefully then we can finally merge this. Sorry this has taken so long!

Comment (briochemc) [reply test/errorshow.jl:294]: @JeffBezanson Are you sure about the spaces after commas? If we are reverting the output of `summary` for tuples to use `typeof`, then the output has no commas, right? E.g., with Julia v1.5.1:
```julia
julia> typeof((1,2,3))
 Tuple{Int64,Int64,Int64} 
```
Or am I missing something?

Comment (fredrikekre) [reply test/errorshow.jl:294]: https://github.com/JuliaLang/julia/pull/37085

Comment (briochemc) [reply test/errorshow.jl:294]: OK I was not aware of that 😅 will change it!

Comment (briochemc) [reply test/errorshow.jl]: OK Done! 🙂

Comment (briochemc) [reply test/errorshow.jl]: @JeffBezanson is it OK this time?

**Comment by briochemc** [comment]
(Gentle bump!)

**Pull request merged**